src/Consolonia.Themes/Templates/Controls/MenuFlyoutPresenter.axaml

```diff
@@ -5,8 +5,11 @@
                   TargetType="MenuFlyoutPresenter">
         <Setter Property="BorderThickness"
                 Value="1" />
-        <Setter Property="BorderBrush"
-                Value="{DynamicResource ThemeBorderBrush}" />
+        <Setter Property="BorderBrush">
+            <Setter.Value>
+                <console:LineBrush Brush="Gray" LineStyle="Edge" />
```

Comment:
We should not use custom colors. Is it something can be replaced by existing color?

Reply:
you are right, this was left-over